test/integration/004_compare_snapshot_test/test_compare_snapshot.py

```diff
@@ -0,0 +1,21 @@
+from test.integration.base import DBTIntegrationTest
+import os
+
+
+class CompareTest(DBTIntegrationTest):
```

Comment:
This seems much nicer now! :)

Reply:
I was introduced to dbt at the stage where `dbt archive` did not exist...so you can just imagine the look on my face when I saw `archive` as a subcommand and I was like "what is this sorcery?!?!" 🙃 Then I gladly found this page that I believe Drew may have left as a nice joke https://docs.getdbt.com/docs/building-a-dbt-project/archival/

Reply:
hahahahaha I love it